### PR TITLE
fix(react-schema-renderer): add try catch in react-schema-renderer shared/expression 

### PR DESCRIPTION
**Pull request by laughing-pic-zhu**
## question
In some scenes, complieExpression may throw and the page will hang up

## What have you changed?
in react-schema-renderer shared/expression ,i add try catch in the complieExpression function

### CODE REVIEW

File: packages/react-schema-renderer/src/shared/expression.ts [hunk 1]
```diff
@@ -18,11 +18,16 @@ export const complieExpression = <Source = any, Context = any>(
...
-      const vars = Object.keys(context || {})
-      const params = vars.map(key => context[key])
-      return new Function(...vars, `return (${matched[1]});`)(...params)
+      try {
+        const matched = source.match(ExpRE)
+        if (!matched) return source
+        const vars = Object.keys(context || {})
+        const params = vars.map(key => context[key])
+        return new Function(...vars, `return (${matched[1]});`)(...params)
+      }catch (e){
```
Comment (janryWang) [inline packages/react-schema-renderer/src/shared/expression.ts:27]: 代码没格式化？

Comment (laughing-pic-zhu) [reply packages/react-schema-renderer/src/shared/expression.ts:27]: 多了缩进符，刚删掉了，再看看可以吗？

Comment (janryWang) [reply packages/react-schema-renderer/src/shared/expression.ts:27]: 还是没格式化呀，讲道理应该是`} catch`，你没安装prettier？

Comment (laughing-pic-zhu) [reply packages/react-schema-renderer/src/shared/expression.ts]: 依赖都下载了的，不知道为啥没触发，已经手动加空格了。

**Pull request merged**